src/main/command/leveling/leaderboards.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+import sqlite3
+
+import discord
+
+from command.command import Command, Parameter, Category
+
+
+class Leaderboards(Command):
+    def __init__(self):
+        super().__init__("leaderboard", "Shows the levels leaderboard", ["leaderboards", "lb", "lboard"], [
+            Parameter("page")], category=Category.LEVELING)
+
+    async def change_page(self, message: discord.Message, page: int, client: discord.Client):
+        sqlite_client = sqlite3.connect('bot_database.db')
+        data_list = sqlite_client.execute('''SELECT ID, LEVEL, XP FROM LEVELS ORDER BY XP DESC LIMIT 3 OFFSET :offset''', {"offset": page}).fetchall()
+
+        lb_list = ''
+        for data in data_list:
+            user = data[0]
+            level = data[1]
+            total_xp = data[2]
+            lb_list += f"<@{user}> | Level: {level} | Total XP: {total_xp}\n"
+
+        embed = discord.Embed(title="Leaderboard", description=lb_list)
+
+        msg = await message.edit(embed=embed)
+        await self.leaderboard_pages(msg, page, client)
+
+    async def leaderboard_pages(self, message: discord.Message, page: int, client: discord.Client):
+        change_to = ''
+
+        def check(reaction, user):
+            print('aaaaaaaaaaa')
```

Comment:
```suggestion
```
Better remove line 34 from the master version, as it's in for debug purposes and will spam the terminal.